source/stylesheets/refills/_animate.scss

```diff
@@ -0,0 +1,78 @@
+$base-accent-color: #477dca !default;
+
+.animated {
+  @include animation-duration(1s);
+  @include animation-fill-mode(both);
+}
+
+.animated.infinite {
+  @include animation-iteration-count(infinite);
+}
+
+.animated.alternate {
+  @include animation-direction(alternate);
+}
+
+.animated.iteration {
+  @include animation-iteration-count(2);
+}
+
+// Tweak the keyframes below. Or swap them with another animation pattern at 
+// https://github.com/daneden/animate.css/tree/master/source
+
+@-webkit-keyframes zoomOut {
+  0% {
+    opacity: 1;
+  }
+
+  50% {
+    opacity: 0;
+    -webkit-transform: scale3d(.3, .3, .3);
+            transform: scale3d(.3, .3, .3);
```

Comment:
Line should be indented 4 spaces, but was indented 12 spaces<br>`.3` should be written with a leading zero as `0.3`